examples/using-css-modules/src/styles/another-page.module.css

```diff
@@ -8,9 +8,10 @@
   font-size: 10rem;
   line-height: 1.5;
   text-decoration: none;
-  &:hover {
-    text-decoration: underline;
-  }
+}
+
+.link:hover {
```

Comment:
is this change necessary?

Reply:
Yes. In the previous code, the style was not enabled.